salt/states/dockerio.py

```diff
@@ -108,13 +108,17 @@
 
 from __future__ import absolute_import
 import functools
+import inspect
 import logging
 
 # Import salt libs
+from salt.exceptions import CommandExecutionError, SaltInvocationError
 from salt.ext.six import string_types
 import salt.utils
 import salt.ext.six as six
 
+import docker
```

Comment:
This should just put a stack trace in the log, but not crash the state run, if it crashes the state run that needs to be fixed.
but yes, this needs to be gated